app/models/vote.rb

```diff
@@ -221,7 +221,8 @@ def user_weight
   # Now we are free to change the implementation later.
   def anonymous?
     (user.votes_anonymous == :no) ||
-      (user.votes_anonymous == :old && updated_at > Time.parse(MO.vote_cutoff))
+      (user.votes_anonymous == :old &&
+                               updated_at > Time.zone.parse(MO.vote_cutoff))
```

Comment:
This indentation is misleading.  I'm pretty sure `==` has higher precedence that `&&`, so I believe this is interpreted as:
`((user.votes_anonymous == :old) && (updated_at > Time.zone.parse(MO.vote_cutoff)))` but this indentation implies that it's interpreted as `(user.votes_anonymous == (:old && update_at > Time.zone.parse(MO.vote_cutoff)))` which I pretty sure is not what is intended.

Reply:
Quite right. I'll fix it.